scalableminds/webknossos
# Transfer user bounding box from volume to hybrid tracing

- I checked the other values and they don't get overwritten or the frontend preserves them

### Steps to test:
- create volume tracing with user bb
- make tracing hybrid
- user bb should still exist

### Issues:
- fixes #4050 
------
